docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1687,6 +1687,50 @@ <h1>Samza Configuration Reference</h1>
                     </td>
                 </tr>
 
+                <tr>
+                    <td class="property" id="stores-changelog-max-message-size-bytes">stores.<span class="store">store-name</span>.changelog.max.message.size.bytes</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        This property sets the maximum size of the messages allowed in the changelog.
+                        The maximum allowed value and the default value both are 1 MB.
```

Comment:
Why is the max allowed value 1MB? If the changelog supports a larger value, it seems like it would be good to allow that.

Reply:
Changed to 5MB.

Reply:
I wasn't referring to the 1MB number specifically. It's fine as 1MB or 5MB.
I was referring to the language used of "maximum allowed value". Why is there a maximum allowed value for this config?

Reply:
Removed the comment about the maximum allowed value.